superset-frontend/src/explore/controls.jsx

```diff
@@ -280,6 +280,7 @@ export const controls = {
       ['PT5M', '5 minutes'],
       ['PT30M', '30 minutes'],
       ['PT1H', '1 hour'],
+      ['PT2H', '2 hours'],
```

Comment:
do you think these should be plural or singular? I see both being used (`5 minutes` but also `6 hour`). Maybe we can standardize while you're in here?

Reply:
I saw the inconsistencies as well and thought it safest to leave them alone for now, but agree that it would be nice to fix them. Do you mind if I put up a separate or for that?

Reply:
I'm not familiar with how these strings are used, but should we be translating these with `tn()`?

Reply:
a separate PR is good, thanks!

Reply:
That's a good point @villebro. I'll add some translations in the next PR, too. thanks!